CryptoPkg/Test/UnitTest/Library/BaseCryptLib/BaseCryptLibUnitTestApp.inf

```diff
@@ -60,8 +60,14 @@
   gEfiCryptoPkgTokenSpaceGuid.PcdCryptoServiceHmacSha256SetKey            ## CONSUMES
   gEfiCryptoPkgTokenSpaceGuid.PcdCryptoServiceHmacSha256Update            ## CONSUMES
   gEfiCryptoPkgTokenSpaceGuid.PcdCryptoServiceHmacSha256Final             ## CONSUMES
+  gEfiCryptoPkgTokenSpaceGuid.PcdCryptoServiceHmacSha384New               ## CONSUMES

```

Comment:
nit: mu change tags missing from this file.

Reply:
Updated.